packages/app/src/app/pages/Sandbox/Editor/Workspace/Chat/index.tsx

```diff
@@ -54,76 +65,93 @@ export const Chat: React.FC = () => {
+              const { color, name, metadata } = messageData(message);
+              return (
+                <Element key={message.date}>
+                  {isNotSameUser(message, i) && (
+                    <Stack
+                      paddingTop={2}
+                      marginBottom={2}
+                      align="center"
+                      gap={2}
+                    >
+                      <Avatar
+                        color={color}
+                        alt={metadata.username}
+                        src={metadata.avatarUrl}
+                      />
+                      <Text block weight="bold">
+                        {name}
+                      </Text>
+                    </Stack>
+                  )}
+                  <Text
+                    block
                     style={{
-                      color,
-                      fontWeight: 600,
-                      marginBottom: '0.25rem',
-                      marginTop: '0.5rem',
+                      wordBreak: 'break-word',
                     }}
+                    marginBottom={2}
                   >
-                    {name}
-                    {currentUserId === message.userId && ' (you)'}
-                    {!metadata && ' (left)'}
-                  </div>
-                )}
-                <div
-                  style={{
-                    color: 'rgba(255, 255, 255, 0.7)',
-                    fontWeight: 400,
-                    marginBottom: '.25rem',
-                  }}
-                >
-                  {message.message.split('\n').map(m => (
-                    <span key={m}>
-                      {m}
-                      <br />
-                    </span>
-                  ))}
-                </div>
-              </div>
-            );
-          })
-        ) : (
-          <div
-            style={{
-              fontStyle: 'italic',
-              color: 'rgba(255, 255, 255, 0.5)',
-            }}
-          >
-            No messages, start sending some!
-          </div>
-        )}
-      </Messages>
-      <AutosizeTextArea
-        useCacheForDOMMeasurements
-        value={value}
-        onChange={handleChange}
-        placeholder="Send a message..."
-        style={{
-          width: '100%',
-          minHeight: height,
-          marginTop: '0.5rem',
-        }}
-        onKeyDown={handleKeyDown}
-        onHeightChange={setHeight}
-      />
-    </Container>
+                    {message.message.split('\n').map(m => (
+                      <span key={m}>
+                        {m}
+                        <br />
+                      </span>
+                    ))}
+                  </Text>
+                </Element>
+              );
+            })
+          ) : (
+            <Text variant="muted">No messages, start sending some!</Text>
+          )}
+        </Messages>
+        <Element marginTop={4}>
+          <Textarea
```

Comment:
the autogrow doesnt reset 🤔 on it's own. need a way to rerender the textarea after submit. changing the `key` prop would do it, what do we add as the key though?